src/app/views/templates/templates.html

```diff
@@ -59,13 +59,24 @@ <h2 class="page-header">Generate Template</h2>
 <div class="row">
   <div class="col-xs-6">
     <div ng-repeat="(worksheet, data) in $ctrl.getAttributes()">
-      <h3 ng-if="$ctrl.worksheet === 'Workbook'">{{ worksheet }} Worksheet</h3>
+      <div ng-if="$ctrl.worksheet === 'Workbook'">
+        <h3>Workbook</h3>
+        <div class="btn-group-xs">
+          <button type="button" class="btn btn-link" ng-click="$ctrl.selectAll('Workbook')">
+            Select ALL in Workbook
+          </button>
+          <button type="button" class="btn btn-link" ng-click="$ctrl.selectNone('Workbook')">
+            Select NONE in Workbook
+          </button>
+        </div>
+        <h3>{{ worksheet }} Worksheet</h3>
+      </div>
```

Comment:
This should be above the ng-repeat, so it only appears 1x on the page.

I'm also don't think we need the "Worksheet" header.

Reply:
Take away the whole header or just the word "worksheet?"